superset/assets/javascripts/explorev2/components/SelectField.jsx

```diff
@@ -26,6 +26,40 @@ const defaultProps = {
 };
 
 export default class SelectField extends React.Component {
+  constructor(props) {
+    super(props);
+    const options = this.props.choices.map((c) => {
```

Comment:
sounds very reasonable and future proof for if/when we'll need to refresh options